test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
Hmm... ok. I would definitely like any new functionality (especially that which has the potential to break existing functionality) to have unit test coverage before landing. Are you cool with that??